templates/tutorials/ewas-suite/slides.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: introduction_slides
```

Comment:
Do you plan to add slides?

Reply:
Yes, I would like to provide an introduction for the topic